apps/src/p5lab/AnimationPicker/AnimationPickerBody.jsx

```diff
@@ -95,8 +95,12 @@ export default class AnimationPickerBody extends React.Component {
     const scrollWindow = event.target;
     const {currentPage, results, pageCount} = this.state;
     const nextPage = currentPage + 1;
+    const scrollContainerHeight = this.scrollListContainer?.current
+      ? this.scrollListContainer.current.clientHeight
+      : 0;
```

Comment:
will `this.scrollListContainer` ever be falsy? we create this ref in `componentDidMount`, so it should always have a value, even if `this.scrollListContainer.current` hasn't been set yet. double-check me on that though.

readability nit: if the value we're looking for is falsy, we can fallback to a default value using the `||` operator

```javascript
const scrollContainerHeight = this.scrollListContainer.current?.clientHeight || 0;
```

^ should do the same thing